LogMonitor/src/LogMonitor/Utility.h

```diff
@@ -58,9 +61,20 @@ class Utility final
         _In_ const std::wstring& To
     );
 
-    static bool isJsonNumber(_In_ std::wstring& str);
+    static bool isJsonNumber(
+        _In_ std::wstring& str);
+
+    static void SanitizeJson(

```

Comment:
wondering how this change came by?

Reply:
I must have move the parameters to the next line to match the way other functions have been authored in the file.